master/buildbot/process/buildstep.py

```diff
@@ -804,9 +804,9 @@ def addCompleteLog(self, name, text):
         loog.finish()
         self._connectPendingLogObservers()
 
-    def addHTMLLog(self, name, html):
+    def addHTMLLog(self, name, html, type):
```

Comment:
we need to make type optional or set it different so we don't break buildbot's api